### ISSUE TITLE
Introduce -lp as an alias for -LiteralPath to make it easier to pass literal paths

### ISSUE THREAD

**Issue by mklement0** [PowerShell/PowerShell#6732]
Cmdlets such as `Get-ChildItem` / `Get-Item` bind the first positional argument to the `-Path` parameter.

This is convenient on the one hand, because `-Path` supports wildcard expressions, so that you can use commands such as `Get-ChildItem *.txt`

Passing paths meant to be _literal_ paths that way _typically_ works, given that `*` and `?` aren't permissible characters in actual file names; however, `[` and `]` are, and passing literal paths with such characters causes them to be interpreted as wildcards too, potentially resulting in non-matches (e.g., `Get-Item tmp[0].txt` won't match a file literally named `tmp[0].txt`) and even errors.

[While _escaping_ wildcard characters is possible, it is tricky and cumbersome](https://github.com/PowerShell/PowerShell/issues/6714#issuecomment-384353966), which is the primary reason that the `-LiteralPath` parameter was introduced.

Given how common it is to pass literal paths, it would be convenient to be able to shorten:

```powershell
Get-Item -LiteralPath tmp[0].txt
```

to:

```powershell
Get-Item -lp tmp[0].txt  # wishful thinking
```

Note that while using _elastic syntax_ does work - `-l` is currently enough to uniquely identify `-LiteralPath` with `Get-Item` for instance - it is not future-proof (and should never be used in scripts), whereas having an immutable, official parameter-name _alias_ is.

Granted, in scripts it's preferable to use the verbose, original names of parameter names (and to prefer named arguments over positional ones and to never rely on elastic syntax), but given how common it is to pass literal paths, `-lp` is still a desirable convenience, both on the command line and in scripts.

Environment data
----------------

Written as of:

```powershell
PowerShell Core v6.0.2
```

**Comment by kvprasoon** [comment]
@TravisEz13  This issue can be closed as this is implemented via #6770

**Comment by mklement0** [comment]
Thanks for taking this on, @kvprasoon - closing this issue.
